### ISSUE TITLE
Issues for Accessibilty and adding FontAwesome icon in Error Messages

### ISSUE THREAD

**Issue by roopa-rauniyar-roostify** [rjsf-team/react-jsonschema-form#1481]
### Prerequisites

- [ ] I have read the [documentation](https://react-jsonschema-form.readthedocs.io/);

### Description

I Need custom messages with some fontAwesome icon in it and that error messages should be accessible to screen reader.

### Steps to Reproduce
1.Generate form through react json schema.
2.Generate any error in fields.
3.Enable screen readers.

#### Expected behavior
Screen reader should read errors generated while blur from one field to another field if the previous focused field have error. 
Need error as:
![image](https://user-images.githubusercontent.com/51839050/66557166-0d630080-eb6f-11e9-95d7-2cf96f9d9eff.png)

Html To be generated : 
Screen reader does not read the error messages on blur .It reads only when someone tap on the error message. 
![image](https://user-images.githubusercontent.com/51839050/66557455-8a8e7580-eb6f-11e9-9ce5-9cca05160cd5.png)
 
As you can see the div which is highlighted grey have some aria attributes and role attribute i want to append that attribute and wrap that div to all error messages.

#### Actual behavior
In current Error as:
![image](https://user-images.githubusercontent.com/51839050/66556960-c2e18400-eb6e-11e9-9919-8f048e7f53b6.png)
the HTML generated is:
![image](https://user-images.githubusercontent.com/51839050/66556995-d1c83680-eb6e-11e9-8c38-93d063799589.png)

### Version

"react-jsonschema-form": "^1.7.0"

**Comment by KKS1** [comment]
@roopa-rauniyar-roostify as we discussed, custom Field Template is the way to go for achieving this, as an interim solution. Long term, I would love to have an ErrorFieldTemplate customization exposed from react json schema forms. Let me know your thoughts @epicfaace/ anyone else facing this.

**Comment by epicfaace** [comment]
Yes, ErrorFieldTemplate is a great idea and something that it would be nice to support in rjsf.
